backend/RayTracing/Classes/Webservice/endpoint.py

```diff
@@ -23,6 +23,9 @@ def raytrace(request):
         img = raytracer.startRayTracing()
 
         plt.imsave(response, img)
+    elif(request.method == "OPTIONS"):
```

Comment:
Why did you have to add this?

Reply:
Thank you for the hint, is fixed in the next commit :)